app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/CreateStudent.php

```diff
@@ -311,9 +311,10 @@ public function form(Form $form): Form
 
     protected function afterCreate(): void
     {
-        /** Student $student */
+        /** @var Student $student */
         $student = $this->getRecord();
 
+        $student->created_at_source = now();
```

Comment:
I beleive Laravel sets `updated_at` during create as well right? So do we want to replicate that here too?

Reply:
yeah sure